hw/ip/hmac/data/hmac_testplan.hjson

```diff
@@ -64,12 +65,5 @@
       milestone: V2
       tests: ["hmac_stress_all"]
     }
```

Comment:
I did not add stress_all test to the common testplan, because I think different modules might include different tests in the stress_all test. If all the modules are excluding the same tests (perhaps CSR tests?), we can consider adding the stress_all test description to the common testplan.